src/components/Login/Login.js

```diff
@@ -15,6 +15,16 @@ class Login extends Component {
     };
   }
 
+  componentDidMount() {
+    this.listener = this.props.firebase.auth.onAuthStateChanged(
```

Comment:
No need to listen the onAuthStateChanged.

you can just check if the AuthContext has authUser set.